### ISSUE TITLE
[PHP] FQN for return type

### ISSUE THREAD

**Issue by maarsson** [20Tauri/DoxyDoxygen#171]
Using the setting `"php_generate_fully_qualified_name": true` it generates docblock with fully qualified parameters, as expected. However the return type is now FQN, only the short class name.
```php
    /**
     * { function_description }
     *
     * @param \App\Models\Order $order 
     *
     * @return Order ( description_of_the_return_value )
     */
    public function refresh(Order $order): Order
    {
    }
```
In this example it should be `@return \App\Models\Order ( description_of_the_return_value )`

**Comment by 20Tauri** [comment]
Thank you for your report, this case has been corrected and will be included in the next version.

**Comment by 20Tauri** [comment]
Fixed in 1.0.1 (evolution branch only)
